adhocracy4/comments_async/serializers.py

```diff
@@ -148,9 +148,9 @@ class CommentListSerializer(CommentSerializer):
 
     def get_comment(self, obj):
```

Comment:
There is no need to change this method as the model does have the empty text after the migration. You should even be able to remove the whole method as we do not need to overwrite anything anymore.

Reply:
I'll definitely remove it then :). Sorry about the chaotic order - I admit that I was afraid of rebasing yesterday. Won't happen again...